content/en/microservices.md

```diff
@@ -1,20 +1,17 @@
 ---
-title: Microservices
+title: Microservices Architecture 
```

Comment:
Let's change the filename as well. 😊
(The filename update will result the change of term's URL too.)

Reply:
Ok, but we'll need to create a redirect. Otherwise, if anyone linked to it, we'll break it. Do you know how to do that?

Reply:
Redirections can be configured by defining redirection rules in `static/_redirects` file (e.g., [the k/w one](https://github.com/kubernetes/website/blob/main/static/_redirects)).

I performed file renaming and redirection rule addition, and pushed as new commit. 😊

We can check that
https://deploy-preview-1549--cncfglossary.netlify.app/microservices/
redirects to
https://deploy-preview-1549--cncfglossary.netlify.app/microservices-architecture/ .

Reply:
Thank you!